src/zimscraperlib/zim/creator.py

```diff
@@ -209,37 +224,56 @@ def start(self):
             self._log_metadata()
 
         if not all(self._metadata.get(key) for key in MANDATORY_ZIM_METADATA_KEYS):
-            raise ValueError("Mandatory metadata are not all set.")
+            missing_keys = [
+                key
+                for key in MANDATORY_ZIM_METADATA_KEYS
+                if not self._metadata.get(key)
+            ]
+            raise ValueError(
+                "Mandatory metadata are not all set. Missing metadata: "
+                f'{",".join(missing_keys)}'
+            )
 
+        # probably not needed if only our methods have been used, but who knows
```

Comment:
Well it's still possible to manipulate the `.metadata` dict directly so it's good to keep